Toolset/libraries/revcommonlibrary.livecodescript

```diff
@@ -137,32 +140,30 @@ on revSetWindowsShellCommand
 end revSetWindowsShellCommand
 
 
-getProp cREVGeneral[pWhichProp] pWhichProfile
-   lock messages
-   local tRevUniqueID
-   put the cREVGeneral["revUniqueID"] of the target into tRevUniqueID
-   unlock messages
-   
-   # OK-2008-04-14 : The long id will return an invalid reference when in edit group mode,
-   # instead the "rugged id" should be used.
-   --put the long id of the target into tTarget
-   
-   # OK-2008-06-02 : Because this script is used in standalones, using revRuggedId will break this
-   # because its located in revBackscript (which is not included in standalones). As we don't require a
-   # rugged id in standalones anyway, just revert to the old behavior in standalone environment.
+getProp revProfile
+  return the cREVGeneral["profile"] of the target
+end revProfile
+
+getProp cREVGeneral[pWhichProp]
    local tTarget
    if the environment is "development" then
+      # OK-2008-04-14/OK-2008-06-02 : The long id will return an invalid reference
```

Comment:
@bwmilby this is a bit odd... just using `the target` below instead of `tTarget` should work in both cases as `the target` is a direct object reference. Indeed in both cases here `the target` is clearly a valid object because otherwise you could not get the long id of it. It should also be marginally faster as we would not be getting the long id pr revRuggedId string and then parsing it again to get an object ref.

Reply:
Note that `the target` can be both a function to get the name of the object and a chunk destination type to directly reference the object (like `me`). Here we are using it as a chunk destination type in all cases.

Reply:
I did not make any functional changes here, so I would need to do some research on this.  It is interesting that the setProp handler does not use the rugged id.

Reply:
@bwmilby It could just be that at one point someone decided that getting the long id and putting it into tTarget was a good idea (not sure why) and later someone fixed the case that it didn't work in edit group mode.